roles/edpm_frr/tasks/install.yml

```diff
@@ -0,0 +1,62 @@
+---
+# Copyright 2023 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Create persistent directories
+  ansible.builtin.file:
+    path: "{{ item.path }}"
+    state: directory
+    setype: "{{ item.setype }}"
+    mode: "{{ item.mode }}"
+  with_items:
+    - { 'path': /var/log/containers/frr, 'setype': container_file_t, 'mode': '0750' }
+    - { 'path': /var/lib/config-data/ansible-generated/frr, 'setype': container_file_t, 'mode': '0750' }
+    - { 'path': /run/frr, 'setype': container_file_t, 'mode': '0755' }
+    - { 'path': /var/log/containers/ovn-bgp-agent, 'setype': container_file_t, 'mode': '0750' }
+    - { 'path': /var/lib/config-data/ansible-generated/ovn-bgp-agent, 'setype': container_file_t, 'mode': '0750' }
+
+- name: Ensure /run/frr is present upon reboot
+  ansible.builtin.copy:
+    dest: /etc/tmpfiles.d/run-frr.conf
+    content: |
+      d /run/frr 0755 root root - -
+
+- name: Gather SELinux fact if needed
+  when:
+    - ansible_facts.selinux is undefined
+  ansible.builtin.setup:
+    gather_subset:
+      - "!all"
+      - "!min"
+      - "selinux"
+
+- name: Create fcontext entry for frr base directory
```

Comment:
we don't need this fcontext because /var/lib/config-/data is labeled with container_file_t by openstack-selinux.

Reply:
ok, I'll remove it